pipeline/models/utils/scoring.py

```diff
@@ -0,0 +1,86 @@
+from sklearn.metrics import mean_absolute_error, mean_squared_error, explained_variance_score, r2_score  # type: ignore
+import numpy as np
+import pandas as pd  # type: ignore
+from bisect import bisect_left
+
+SPEED_LIMITS = [15, 30, 40, 50, 60, 70, 80, 90, 100, 110, 120, 130]
+
+
+def find_closest_speed_limit(speed: float) -> float:
+    """
+    Returns closest value in speed_limits.
+    If two numbers are equally close, return the smallest number.
```

Comment:
Tilføj grunden dertil, det føles arbitrært som det er skrevet nu. 
Men det er grundlæggende fordi det er sikkerhedsmæssigt bedre at køre langsommere og man kommer ikke i bødeproblemer ift. at køre for stærkt